### DISCUSSION TITLE
<picture> wrapper working on computer, but not mobile

### DISCUSSION THREAD

**Discussion by bnbinder** [community/community#55364] [Mobile]
### Select Topic Area

Bug

### Body

I wanted to have an image on my github readme to be different whether the user was using light or dark mode. This was the best and only working solution I could find. It works when viewing my profile from a computer, but not when viewing from mobile. When on mobile, all I get is a little no image icon surrounded by borders. does ```<picture>``` not work on mobile? Am I using it incorrectly? Thanks in advance.

```
<p align="center">
  <picture>
    <source media="(prefers-color-scheme: light)" srcset="light.jpg"  align="center">
    <img width=400>
  </picture>
</p>
<p align="center">
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="dark.jpg"  align="center">
    <img width=400>
  </picture>
</p>
```

this is what i get on mobile
![IMG_8852](https://github.com/community/community/assets/92772910/4a647488-56a4-4079-91ae-2b21ccfc16a1)

**Comment by Tukmoola** [comment]
When you use Dictation, your device will indicate in Keyboard Settings if your voice inputs and transcripts for general text Dictation (for example, composing messages and notes, but not dictating in a search box) are processed on your device and not sent to Siri servers. Otherwise, the things you dictate are sent to and processed on the server.

When you use Siri and Dictation, your device will send other Siri Data, such as:

• Contact names, nicknames, and relationships (for example, “my dad”), if you set them up in your contacts
• Music and podcasts you enjoy
• Names of your and your Family Sharing members’ devices
• Names of accessories, homes, scenes, shared home members in the Home app, and Apple TV user profiles
• Labels for items, such as people names in Photos, Alarm names, and names of Reminders lists
• Names of apps installed on your device and shortcuts you added through Siri

- [x] Your requests are associated with a random identifier, not your Apple ID.```
```
duri```
> thn`bbd`
```
```
```

**Comment by Seunfola** [comment]
I have a similar issue but mine was using markdown to display picture it is not work.

![]() Just not displaying the pictures i do not know why .

**Comment by rajuahmed78** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> I wanted to have an image on my github readme to be different whether the user was using light or dark mode. This was the best and only working solution I could find. It works when viewing my profile from a computer, but not when viewing from mobile. When on mobile, all I get is a little no image icon surrounded by borders. does ```<picture>``` not work on mobile? Am I using it incorrectly? Thanks in advance.
> 
> ```
> <p align="center">
>   <picture>
>     <source media="(prefers-color-scheme: light)" srcset="light.jpg"  align="center">
>     <img width=400>
>   </picture>
> </p>
> <p align="center">
>   <picture>
>     <source media="(prefers-color-scheme: dark)" srcset="dark.jpg"  align="center">
>     <img width=400>
>   </picture>
> </p>
> ```
> 
> this is what i get on mobile
> ![IMG_8852](https://github.com/community/community/assets/92772910/4a647488-56a4-4079-91ae-2b21ccfc16a1)
> 

***{Use mobile vivo 1933}***

**Reply by bnbinder** [reply]
i have no idea what "mobile vivo 1933" is. can you please elaborate on this? thank you.
